packages/main/src/Table.js

```diff
@@ -36,9 +36,9 @@ const metadata = {
 	slots: /** @lends sap.ui.webcomponents.main.Table.prototype */ {
 
 		/**
-		 * Defines the <code>ui5-table</code> rows.
+		 * Defines the component rows.
 		 * <br><br>
-		 * <b>Note:</b> Use <code>ui5-table-row</code> for the intended design.
+		 * <b>Note:</b> Use the component for the intended design.
```

Comment:
Here I guess we need to leave `<code>ui5-table-row</code>` as the idea is to inform which component has to be used as a child